tuf/utils/x509.go

```diff
@@ -409,6 +409,17 @@ func KeyToPEM(privKey data.PrivateKey, role string) ([]byte, error) {
 			"role": role,
 		}
 	}
+	if gun != "" {
+		headers = map[string]string{
```

Comment:
Totally non-blocking - can we just set the header value for the given key in two if blocks, rather than redeclaring it in three if blocks?